packages/@blockprotocol/graph/src/types/ontology.ts

```diff
@@ -1,7 +1,10 @@
-import { BaseUri } from "@blockprotocol/type-system/dist/cjs-slim/index-slim";
-import { validateBaseUri } from "@blockprotocol/type-system/wasm/type-system";
+import { BaseUri, validateBaseUri } from "@blockprotocol/type-system/slim";
+
+export * from "./ontology/data-type.js";
```

Comment:
I had opened a comment about this but force-pushed over it or something I think 😅 

I'm not sure why this is required, I haven't seen it in other packages, I think it might be because of how the `dist` is generated and the module is ESM. I encountered an issue when _using_ this package (not building it or through `tsc` which is frustrating) in `mock-block-dock`, I don't remember the error unfortunately, but I think it was about ESM and it explicitly told me I needed file endings. There were also file endings in some of the code already, but inconsistently, which confused me even more.

Perhaps @kachkaev has an idea.

Reply:
You need file extensions on relative import statements in ESM packages, i.e. those with `type: "module"` - [docs](https://nodejs.org/api/esm.html#import-specifiers)